src/dcqc/target.py

```diff
@@ -65,10 +64,30 @@ def get_file_type(self) -> FileType:
             raise NotImplementedError(message)
         return file_type
 
-    @wraps(File.stage)
     def stage(
         self, destination: Optional[Path] = None, overwrite: bool = False
     ) -> list[Path]:
+        """Create local copy of local or remote file.
+
+        A destination is not required for remote files; it
+        defaults to a temporary directory.
+        Local files aren't moved if a destination is omitted.
+
+        Args:
+            destination: File or folder where to store the file.
+                Defaults to None.
+            overwrite: Whether to ignore existing file at the
+                target destination. Defaults to False.
+
+        Raises:
+            ValueError: If the parent directory of the
+                destination does not exist.
+            FileExistsError: If the destination file already
+                exists and ``overwrite`` was not enabled.
+
+        Returns:
+            The path of the local copy.
+        """
```

Comment:
You can ignore this. I tried to avoid docstring duplication with `wraps()`, but it was causing an error in VS Code because of the parameters. It's unrelated to the PR, but I deemed it small enough to sneak in.